context/AppReducer.js

```diff
@@ -0,0 +1,26 @@
+export const initialState = {
+  chain: {
+    nodeAddress: process.env.NEXT_PUBLIC_NODE_ADDRESS,
+    denom: process.env.NEXT_PUBLIC_DENOM,
+    displayDenom: process.env.NEXT_PUBLIC_DISPLAY_DENOM,
+    displayDenomExponent: parseInt(process.env.NEXT_PUBLIC_DISPLAY_DENOM_EXPONENT, 10),
+    gasPrice: process.env.NEXT_PUBLIC_GAS_PRICE,
+    chainId: process.env.NEXT_PUBLIC_CHAIN_ID,
+    addressPrefix: process.env.NEXT_PUBLIC_ADDRESS_PREFIX,
```

Comment:
Could you add the `NEXT_PUBLIC_EXPLORER_LINK_TX` from #23?

Reply:
ok rebased and added this to the context object.